lib/physiqual/sessions/token_authorized_session.rb

```diff
@@ -12,6 +12,7 @@ def get(path, params = {})
         result = HTTParty.get(full_url_for(path),
                               query: params,
                               headers: @header)
+        fail(Errors::UnexpectedHttpResponseError, result.response.to_s) unless result.response.code == '200'
```

Comment:
Volgens mij is het hier netter om ook 202 e.d. Te accepteren (alles tussen de 2 en 300 is valid volgens mij), maar dit gaat ook wel goed.